### DISCUSSION TITLE
Initial d arcade stage ver 2 controller problems

### DISCUSSION THREAD

**Discussion by akumaisdrifting** [flyinghead/flycast#1130] [Q&A]
So I'm not sure if it's an issue with the emulator, or if it is the game but I'm having a problem with the controls: the thing is that the game for some reason is having some kind of "drift" no matter what kind of controller I'm using I can't control where I want to go exactly, like if I want to select a car brand, or the game mode, and who I want race against. I've tried with a keyboard, a controller, even a mouse and is having the same problem, and the weird thing is that only happens when I have to select something with the thumbstick left or right, when I'm in a race the controls are perfect.

**Comment by flyinghead** [comment]
It is a sensitivity issue. In the menus this game is not expecting large deflections from its wheel. So using the keyboard (instant 0-100%) is almost impossible. And the thumbstick must be moved *very* lightly.
Menu items are selected by a given deflection angle on the wheel, which is easy to hold if you don't move it, but harder with a thumbstick since there's a centering counter-force.
